cli-typescript/src/utils/cmdActions/deployAction.ts

```diff
@@ -41,7 +41,11 @@ const deployAction = async (
 
     const { signer } = await getProjectSigner(symbol);
 
-    const cm = new ContractManager(mergedConfig.chainId, signer);
+    const cm = new ContractManager(
+      mergedConfig.chainId,
+      signer,
+      symbol.toLowerCase(),
```

Comment:
Should we parse this on the `ContractManager` class side too?

Reply:
Yes, thanks for the feedback.